components/nova/aio-values.yaml

```
@@ -18,6 +18,12 @@ endpoints:
       name: rabbitmq-server
     hosts:
       default: rabbitmq-nodes
+  compute:
+    port:
+      api:
```

```suggestion
      osapi:
```

I think nova uses that.

---

https://opendev.org/openstack/openstack-helm/src/commit/8e5ad85fd09983e07ddc87de76e480b4ceca9120/nova/values.yaml#L1880